docs/pubsub/publisher/index.rst

```diff
@@ -59,7 +59,7 @@ Batching
 --------
 
 Whenever you publish a message, a
-:class:`~.pubsub_v1.publisher.batch.thread.Batch` is automatically created.
+:class:`~.pubsub_v1.publisher._batch.thread.Batch` is automatically created.
```

Comment:
we'll need to update this paragraph (and the other one below) to just say something like "Whenever you publish a message, the publisher will automatically batch the messages over a small time window to avoid making too many separate requests to the service and increase throughput."